run_unittests.py

```diff
@@ -13,12 +13,17 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+<<<<<<< dbcbf19ecea9d07d264dbbc1cd87ab22393be8a7
 import stat
+=======
+import shlex
+>>>>>>> Use CompilerArgs for generation of compile commands
 import unittest, os, sys, shutil, time
 import subprocess
 import re, json
 import tempfile
 from glob import glob
+import mesonbuild.compilers
```

Comment:
__[flake8]__
* [E402] module level import not at top of file

<!-- Comment by [SideCI](https://sideci.com) -->